src/views/Lessons/Module1.tsx

```diff
@@ -51,13 +89,30 @@ class Page1 extends Module1 {
 }
 
 class Page2 extends Module1 {
+    constructor() {
+        super(
+            "https://res.cloudinary.com/dn8rdavoi/video/upload/v1707854682/kids-go-server-COMBINED-audio-slices/slice_audio2_pcjrth.wav",
+        );
+        console.log("page 2 constructor");
+    }
+
     text(): JSX.Element | Array<JSX.Element> {
-        return (
+        return [
+            <button key="playButton" onClick={this.playAudio}>
```

Comment:
If we have a play button, we probably want it to switch to "pause" or "stop" or something like that when the audio is actively playing

Reply:
Yep, agreed, this was something Paul and I discussed.  

I'd like to figure out how to use the useState() hook because I'm not familiar with the equivalent class component code .

Do you have any suggestions for this?

Reply:
Feel free to refactor to hooks if you've got a nice way of doing that, the class based stuff is still there because of inheritance stuff.

However, the class based stuff has state as well, you don't use `useState` instead you just stuff things in `this.state = { "foo": "bar" }` and update with `this.setState({"foo": "car"})`, there's still a few class based components floating around in the online-go.com code base if the reference helps: https://github.com/online-go/online-go.com/blob/devel/src/components/ModNoteModal/ModNoteModal.tsx#L39